hw/ip/clkmgr/data/clkmgr.sv.tpl

```diff
@@ -322,7 +322,11 @@
   );
 
   prim_clock_gating #(
-    .NoFpgaGate(1'b1)
+% if k == "clk_io_div4_peri" or k == "clk_usb_peri":
```

Comment:
PR for another day..but do you think it makes sense to specify the FPGA global vs local clock buffer in the hjson itself? 
Then we can take that info and automatically generate the appropriate parameter options.

Reply:
Yes, that would make a lot of sense.